editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/l10n/messages.properties

```diff
@@ -1,69 +1,78 @@
+newProjectWizardRuntimePageRuntimeDescription = Select a target runtime you want to deploy to from the list or use the New button to create one.
+newProjectWizardRuntimePageRuntimeNewButtonLabel = New
+newProjectWizardRuntimePageRuntimeNewButtonDescription = Use this button to create a runtime definition.
+newProjectWizardRuntimePageCamelGroupLabel = Camel Version
+newProjectWizardRuntimePageCamelLabel = Select the Camel version for your new project
+newProjectWizardRuntimePageCamelDescription = Select a Camel version from our list of supported versions or enter a different one.
+newProjectWizardRuntimePageNoRuntimeSelectedLabel = No Runtime selected
+newProjectWizardRuntimePageCamelVersionsDontMatchWarning = \nThe selected target runtime contains a different version of Apache Camel. You should align with the version installed in the runtime ({0}) or select a different runtime which supports your selected Camel version. You may continue at your own risk.
+newProjectWizardRuntimePageCamelVersionInvalidWarning = \nThe selected Apache Camel version {0} seems to be unavailable. Possible reasons are a missing internet connection or the Camel version simply does not exist. Please choose a different version.
+newProjectWizardRuntimePageResolveDependencyStatus=Resolving Apache Camel version...
+
+newProjectWizardTemplatePageName = Templates
+newProjectWizardTemplatePageTitle = Advanced Project Setup
+newProjectWizardTemplatePageDescription = Please select how you would like to setup your project.
+newProjectWizardTemplatePageHeadlineLabel = What would you like to do?
+newProjectWizardTemplatePageEmptyProjectLabel = Start with an empty project
+newProjectWizardTemplatePageEmptyProjectDescription = Select this option if you want to start with a blank project.
+newProjectWizardTemplatePageBlueprintDSLLabel = Blueprint DSL
+newProjectWizardTemplatePageBlueprintDSLDescription = Use this option to start with a Blueprint Camel context ready to be deployed to an OSGi container supporting Apache Camel.
+newProjectWizardTemplatePageSpringDSLLabel = Spring DSL
+newProjectWizardTemplatePageSpringDSLDescription = Use this option to start with a Spring Camel context used for Camel on EAP for instance.
+newProjectWizardTemplatePageJavaDSLLabel = Java DSL
+newProjectWizardTemplatePageJavaDSLDescription = Use this option if you want to create your Camel route using Java directly.
+newProjectWizardTemplatePageTemplateProjectLabel = Use a predefined template
+newProjectWizardTemplatePageTemplateProjectDescription = Select this option if you would like to start with one of our predefined project templates.
+newProjectWizardTemplatePageDSLLabel = Select the project type
+newProjectWizardTemplatePageFilterBoxText = type filter text (filters on name, description and keywords)
+UnzipStreamCreator_UnzippingTemplateFileMonitorMessage=Unzipping template files...
+
+stagingRepositoriesPreferencePageDescription=Preferences used when specifying Staging Repositories...
```

Comment:
i would just put : instead of ... at the end.

I would be more precise on why and where it is used, something like:
"Provide ability to specify extra repositories which will be added to Camel project. This is useful when you are working with staging repositories for instance."